### PR TITLE
Replace the random char seed to global rand to avoid collision

### LINKED ISSUE #1728: Large amount of failure of TestNamespace/Concurrent_creating_namespaces in CI
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Version

unstable

### Minimal reproduce step

https://github.com/apache/kvrocks/actions/runs/6062050794/job/16447962006?pr=1712
https://github.com/apache/kvrocks/actions/runs/6062030374/job/16447930241?pr=1630
https://github.com/apache/kvrocks/actions/runs/6062030374/job/16447930290?pr=1630
...

cc @git-hulk 

### What did you expect to see?

-

### What did you see instead?

-

### Anything Else?

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by git-hulk**
#1727 changed the random string causing namespaces test fails.

This fixes #1728

### CODE REVIEW

**Review by mapleFU — DISMISSED** [review]
Aha, so thats `GetCurrentTime` collision to many times...

I'll write a GetUUID64 instead. Lets check this in first

**Comment by mapleFU** [comment]
```go
func RandString(min, max int, typ RandStringType) string {
	return RandStringWithSeed(min, max, typ, rand.Int63())
}
```

The change LGTM, however, would it better to change `RandString` as above? Since `WithSeed` should better have a stable output with the same seed.

**Comment by git-hulk** [comment]
> ```go
> func RandString(min, max int, typ RandStringType) string {
> 	return RandStringWithSeed(min, max, typ, rand.Int63())
> }
> ```
> 
> The change LGTM, however, would it better to change `RandString` as above? Since `WithSeed` should better have a stable output with the same seed.

I changed the seed to rand.Int63. It's better than the time seed.

**Review by mapleFU — DISMISSED** [review]

**Review by enjoy-binbin — DISMISSED** [review]

File: tests/gocase/util/random.go [hunk 1]
```diff
@@ -75,7 +74,8 @@ func RandStringWithSeed(min, max int, typ RandStringType, seed int64) string {
 
 	var sb strings.Builder
 	for ; length > 0; length-- {
-		s := fmt.Sprintf("%c", minVal+int(r.Int31n(int32(maxVal-minVal+1))))
+		// use global rand to avoid collision
```
Comment (mapleFU) [inline tests/gocase/util/random.go]: remove this line?

Comment (git-hulk) [inline tests/gocase/util/random.go]:
```suggestion
```

**Review by mapleFU — APPROVED** [review]

**Pull request merged**